source/receiver.ts

```diff
@@ -127,12 +128,12 @@ function onMessageExternalListener(
 async function prepareResponse(
   message: Message,
   action: "respond" | "forward",
-  meta: MessengerMeta,
+  options: Options,
 ): Promise<unknown> {
   const { type, target, args } = message;
 
   if (action === "forward") {
-    return messenger(type, meta, target, ...args);
+    return messenger(type, options, target, ...args);
```

Comment:
Before: `retry: false` was ignored when the background page forward messages

After: the information is preserved and forwarded messages are not retried